Libplanet/Blockchain/BlockLocator.cs

```diff
@@ -8,6 +9,37 @@ internal class BlockLocator : IEnumerable<HashDigest<SHA256>>
     {
         private readonly IEnumerable<HashDigest<SHA256>> _impl;
 
+        public BlockLocator(
+            Func<long, HashDigest<SHA256>?> indexBlockHash,
+            Func<HashDigest<SHA256>, long> blockHashByIndex,
```

Comment:
How about this? Because the function gets index by block hash.

```suggestion
            Func<HashDigest<SHA256>, long> indexByBlockHash,
```